lib/formfields/admin/domains/formfield.domains_edit.php

```diff
@@ -14,6 +14,28 @@
  * @package    Formfields
  *
  */
+
```

Comment:
please move this logic out of this file as it JUST returns an array and formfields should not include anything like that! (We've also talked about that)

Reply:
Where should it move to? Please make a proposal ;)

Reply:
where the formfield is used, before https://github.com/Froxlor/Froxlor/blob/master/admin_domains.php#L629 you can make all adjustments required

Reply:
I will move it to this location.

Reply:
don't forget customer_domains.php too if you add any options there (`lib/formfields/customer/domains/...`)

Reply:
Do we want customers to access DKIM, too? Customers do not have the rights yet to enable/disable DKIM so I am not sure it is neccessary to show the zonefile additions...

Reply:
But maybe they should **see** then dns-entry they should add to their zone if not managed by that froxlor instance - what if the admin does not have access to the customers DNS? should be visible when activated

Reply:
Thats correct. I will add that.